test/tools/ClientCertificateCheck/Pages/Index.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿@page
+@model IndexModel
+Status: @Model.Status
+Thumbprint: @Model.CertThumbprint
+Subject: @Model.CertSubject
+Subject Name: @Model.CertSubjectName
+Issuer: @Model.CertIssuer
+Issuer Name: @Model.CertIssuerName
+Not After: @Model.CertNotAfter
+Not Before: @Model.CertNotBefore
```

Comment:
GitHub mark absent new line at EOF as error and suggest GitHub community to fix it. We already fixed most of our files in the repo. So please add new line at EOF in all files you change until it is really need empty file.